PHPCompatibility/Tests/sniff-examples/deprecated_magic_autoload.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function __autoload($someclass) {
+    echo 'I am the autoloader - I am deprecated from PHP 7.2 onwards';
+}
+
+class foo {
+    function __autoload($someclass) {
```

Comment:
Are you sure this code works as an autoloader ? I always though the `__autoload()` function had to be defined in the global namespace ?